### ISSUE TITLE
Add !gnews trigger

### ISSUE THREAD

**Issue by adamstirtan** [adamstirtan/ircbot#4]
URL: https://ajax.googleapis.com/ajax/services/search/news?v=1.0&q=Paris%20Hilton

Response:

{"responseData": {"results":[{"GsearchResultClass":"GnewsSearch","clusterUrl":"","content":"\u003cb\u003eParis Hilton\u003c/b\u003e made a bold move during her Ibiza DJ set over the weekend - choosing to play Wonderwall. It was a risky decision, but one that paid off for the blonde beauty as a sea of ravers began swaying to the track. The incident happened as reality \u003cb\u003e...\u003c/b\u003e","unescapedUrl":"http://www.mirror.co.uk/3am/celebrity-news/watch-paris-hilton-play-wonderwall-6108661","url":"http%3A%2F%2Fwww.mirror.co.uk%2F3am%2Fcelebrity-news%2Fwatch-paris-hilton-play-wonderwall-6108661","title":"Watch \u003cb\u003eParis Hilton\u003c/b\u003e play WONDERWALL during her Ibiza DJ set - how did the crowd \u003cb\u003e...\u003c/b\u003e","titleNoFormatting":"Watch Paris Hilton play WONDERWALL during her Ibiza DJ set - how did the crowd ...","location":"","publisher":"Mirror.co.uk","publishedDate":"Tue, 21 Jul 2015 06:30:00 -0700","signedRedirectUrl":"http://news.google.com/news/url?sa\u003dT\u0026ct2\u003dus\u0026fd\u003dS\u0026url\u003dhttp://www.mirror.co.uk/3am/celebrity-news/watch-paris-hilton-play-wonderwall-6108661\u0026cid\u003d52778906062656\u0026ei\u003dYQWvVbCBJPCY2gXJnqPYAQ\u0026usg\u003dAFQjCNFLXtLnV8TaY89sv6tBqZdW33l34g","language":"en","image":{"url":"http://brightcove.vo.llnwd.net/v1/unsecured/media/4221396001/201507/271/4221396001_4365567371001_4365526373001-vs.jpg?pubId\u003d4221396001","tbUrl":"http://t3.gstatic.com/images?q\u003dtbn:ANd9GcTUDgkhWCdX6j9cqRTvAEFGRFSi7UT2M39qSfRDvIL_W1uaoPblmJlb1M-B","originalContextUrl":"http://www.mirror.co.uk/3am/celebrity-news/watch-paris-hilton-play-wonderwall-6108661","publisher":"Mirror.co.uk","tbWidth":80,"tbHeight":45}},{"GsearchResultClass":"GnewsSearch","clusterUrl":"http://news.google.com/news/story?ncl\u003dd8ZuzfQ02W_oUKMRhF43AsuTRh4-M\u0026hl\u003den\u0026ned\u003dus","content":"Remember The Simple Life? It was a gem of a reality show that aired from 2003 to 2007 that followed the wacky hijinks of \u003cb\u003eParis Hilton\u003c/b\u003e and Nicole Richie as they hilariously explored the vast landscape that is the United States. Would Nicole ever \u003cb\u003e...\u003c/b\u003e","unescapedUrl":"http://www.wetpaint.com/2015-07-21-nicole-richie-simple-life-reunion-paris-hilton/","url":"http%3A%2F%2Fwww.wetpaint.com%2F2015-07-21-nicole-richie-simple-life-reunion-paris-hilton%2F","title":"Would Nicole Richie Do a Simple Life Reunion With \u003cb\u003eParis Hilton\u003c/b\u003e? | Wetpaint \u003cb\u003e...\u003c/b\u003e","titleNoFormatting":"Would Nicole Richie Do a Simple Life Reunion With Paris Hilton? | Wetpaint ...","location":"","publisher":"Wetpaint","publishedDate":"Tue, 21 Jul 2015 15:07:30 -0700","signedRedirectUrl":"http://news.google.com/news/url?sa\u003dT\u0026ct2\u003dus\u0026fd\u003dS\u0026url\u003dhttp://www.wetpaint.com/2015-07-21-nicole-richie-simple-life-reunion-paris-hilton/\u0026cid\u003d52778905466553\u0026ei\u003dYQWvVbCBJPCY2gXJnqPYAQ\u0026usg\u003dAFQjCNGisadT3SRvlQptXMNNPnkOI4dyqg","language":"en","image":{"url":"http://az711664.vo.msecnd.net/wetpaint/2015/07/MV5BMjEzNTgyMjY5N15BMl5BanBnXkFtZTYwMTQyMTQ3._V1__SX1261_SY688_-750x522-1437516613.jpg","tbUrl":"http://t2.gstatic.com/images?q\u003dtbn:ANd9GcSoimN6OtBOwGB3ocfVxppu-TBuBYLfLt7-n_WO2g6w_d3h4s90zi7aoPGd","originalContextUrl":"http://www.wetpaint.com/2015-07-21-nicole-richie-simple-life-reunion-paris-hilton/","publisher":"Wetpaint","tbWidth":80,"tbHeight":55},"relatedStories":[{"unescapedUrl":"http://www.dailymail.co.uk/tvshowbiz/article-3169565/Nicole-Richie-33-reveals-two-children-husband-Joel-Madden.html","url":"http%3A%2F%2Fwww.dailymail.co.uk%2Ftvshowbiz%2Farticle-3169565%2FNicole-Richie-33-reveals-two-children-husband-Joel-Madden.html","title":"Nicole Richie reveals two children are enough with husband Joel Madden | Daily \u003cb\u003e...\u003c/b\u003e","titleNoFormatting":"Nicole Richie reveals two children are enough with husband Joel Madden | Daily ...","location":"","publisher":"Daily Mail","publishedDate":"Tue, 21 Jul 2015 07:49:55 -0700","signedRedirectUrl":"http://news.google.com/news/url?sa\u003dT\u0026ct2\u003dus\u0026fd\u003dS\u0026url\u003dhttp://www.dailymail.co.uk/tvshowbiz/article-3169565/Nicole-Richie-33-reveals-two-children-husband-Joel-Madden.html\u0026cid\u003d52778905466553\u0026ei\u003dYQWvVbCBJPCY2gXJnqPYAQ\u0026usg\u003dAFQjCNGbVWdYVOF9wL4IkPxpMpIthzVvqg","language":"en"},{"unescapedUrl":"http://www.heatworld.com/2015/07/nicole-richie-talks-having-more-kids-and-simple-life-reunion","url":"http%3A%2F%2Fwww.heatworld.com%2F2015%2F07%2Fnicole-richie-talks-having-more-kids-and-simple-life-reunion","title":"Nicole Richie talks having more kids and Simple Life reunion | Celeb News \u003cb\u003e...\u003c/b\u003e","titleNoFormatting":"Nicole Richie talks having more kids and Simple Life reunion | Celeb News ...","location":"","publisher":"heatworld","publishedDate":"Tue, 21 Jul 2015 04:26:15 -0700","signedRedirectUrl":"http://news.google.com/news/url?sa\u003dT\u0026ct2\u003dus\u0026fd\u003dS\u0026url\u003dhttp://www.heatworld.com/2015/07/nicole-richie-talks-having-more-kids-and-simple-life-reunion\u0026cid\u003d52778905466553\u0026ei\u003dYQWvVbCBJPCY2gXJnqPYAQ\u0026usg\u003dAFQjCNEOmYDikPOzyTOaXajAfoU_VClOQA","language":"en"},{"unescapedUrl":"http://primetime.unrealitytv.co.uk/the-simple-life-reunion-nicole-richie-speaks-paris-hilton/","url":"http%3A%2F%2Fprimetime.unrealitytv.co.uk%2Fthe-simple-life-reunion-nicole-richie-speaks-paris-hilton%2F","title":"The Simple Life reunion!? Nicole Richie speaks out | Unreality TV","titleNoFormatting":"The Simple Life reunion!? Nicole Richie speaks out | Unreality TV","location":"","publisher":"Unreality TV","publishedDate":"Tue, 21 Jul 2015 08:45:00 -0700","signedRedirectUrl":"http://news.google.com/news/url?sa\u003dT\u0026ct2\u003dus\u0026fd\u003dS\u0026url\u003dhttp://primetime.unrealitytv.co.uk/the-simple-life-reunion-nicole-richie-speaks-paris-hilton/\u0026cid\u003d52778905466553\u0026ei\u003dYQWvVbCBJPCY2gXJnqPYAQ\u0026usg\u003dAFQjCNHUoOFA9Mjrl5QfTUoVS_-6oC8pXA","language":"en"},{"unescapedUrl":"http://www.breathecast.com/articles/nicole-richie-news-fashion-designer-talks-family-life-and-her-career-29854/","url":"http%3A%2F%2Fwww.breathecast.com%2Farticles%2Fnicole-richie-news-fashion-designer-talks-family-life-and-her-career-29854%2F","title":"Nicole Richie News: Fashion Designer Talks Family Life and Her Career : In the \u003cb\u003e...\u003c/b\u003e","titleNoFormatting":"Nicole Richie News: Fashion Designer Talks Family Life and Her Career : In the ...","location":"","publisher":"BREATHEcast","publishedDate":"Tue, 21 Jul 2015 10:37:30 -0700","signedRedirectUrl":"http://news.google.com/news/url?sa\u003dT\u0026ct2\u003dus\u0026fd\u003dS\u0026url\u003dhttp://www.breathecast.com/articles/nicole-richie-news-fashion-designer-talks-family-life-and-her-career-29854/\u0026cid\u003d52778905466553\u0026ei\u003dYQWvVbCBJPCY2gXJnqPYAQ\u0026usg\u003dAFQjCNHN55cP1Flii6plANwobrfHkUMH0A","language":"en"}]},{"GsearchResultClass":"GnewsSearch","clusterUrl":"","content":"The Rich Kids of the Cartel: Mexico drugs lord El Chapo\u0026#39;s blinged-up children show off their fabulous wealth with flash cars, guns, \u003cb\u003eParis Hilton\u003c/b\u003e selfies, and pet lions. Billionaire drugs boss \u0026#39;El Chapo\u0026#39; Guzman is one of the world\u0026#39;s most wanted men \u003cb\u003e...\u003c/b\u003e","unescapedUrl":"http://www.dailymail.co.uk/news/article-3168750/The-Rich-Kids-Cartel-Mexico-drugs-lord-El-Chapo-s-blinged-children-fabulous-wealth-flash-cars-guns-Paris-Hilton-selfies-pet-lions.html","url":"http%3A%2F%2Fwww.dailymail.co.uk%2Fnews%2Farticle-3168750%2FThe-Rich-Kids-Cartel-Mexico-drugs-lord-El-Chapo-s-blinged-children-fabulous-wealth-flash-cars-guns-Paris-Hilton-selfies-pet-lions.html","title":"El Chapo\u0026#39;s children show off their wealth with guns and \u003cb\u003eParis Hilton\u003c/b\u003e selfies \u003cb\u003e...\u003c/b\u003e","titleNoFormatting":"El Chapo\u0026#39;s children show off their wealth with guns and Paris Hilton selfies ...","location":"","publisher":"Daily Mail","publishedDate":"Tue, 21 Jul 2015 01:03:45 -0700","signedRedirectUrl":"http://news.google.com/news/url?sa\u003dT\u0026ct2\u003dus\u0026fd\u003dS\u0026url\u003dhttp://www.dailymail.co.uk/news/article-3168750/The-Rich-Kids-Cartel-Mexico-drugs-lord-El-Chapo-s-blinged-children-fabulous-wealth-flash-cars-guns-Paris-Hilton-selfies-pet-lions.html\u0026cid\u003d52778902756321\u0026ei\u003dYQWvVbCBJPCY2gXJnqPYAQ\u0026usg\u003dAFQjCNE7BHAX_-OxffXGnThmtjwIIetzOA","language":"en","image":{"url":"http://i.dailymail.co.uk/i/pix/2015/07/20/21/2AB4305000000578-0-image-a-39_1437425807809.jpg","tbUrl":"http://t2.gstatic.com/images?q\u003dtbn:ANd9GcTf2ZWoWoMmdHUCElviElwODJfiing_kfvQXAwm1s6wCGFlfMG85nb1bSY","originalContextUrl":"http://www.dailymail.co.uk/news/article-3168750/The-Rich-Kids-Cartel-Mexico-drugs-lord-El-Chapo-s-blinged-children-fabulous-wealth-flash-cars-guns-Paris-Hilton-selfies-pet-lions.html","publisher":"Daily Mail","tbWidth":66,"tbHeight":80}},{"GsearchResultClass":"GnewsSearch","clusterUrl":"http://news.google.com/news/story?ncl\u003ddnm7YOnjZ3vrgdMpPDIFdr3ZDPSIM\u0026hl\u003den\u0026ned\u003dus","content":"Subject in question: \u003cb\u003eParis Hilton\u003c/b\u003e. I want to keep an open mind about her, but she\u0026#39;s not giving me much to work with here. While certainly no one thought her relationship with Afrojack would have directly resulted in her having a go at a DJ career, it\u0026#39;s \u003cb\u003e...\u003c/b\u003e","unescapedUrl":"http://www.youredm.com/2015/07/19/paris-hilton-thinks-shes-proven-her-haters-wrong-in-recent-interview/","url":"http%3A%2F%2Fwww.youredm.com%2F2015%2F07%2F19%2Fparis-hilton-thinks-shes-proven-her-haters-wrong-in-recent-interview%2F","title":"\u003cb\u003eParis Hilton\u003c/b\u003e Thinks She\u0026#39;s Proven Her Haters Wrong in Recent Interview | Your EDM","titleNoFormatting":"Paris Hilton Thinks She\u0026#39;s Proven Her Haters Wrong in Recent Interview | Your EDM","location":"","publisher":"Your EDM","publishedDate":"Sun, 19 Jul 2015 14:00:00 -0700","signedRedirectUrl":"http://news.google.com/news/url?sa\u003dT\u0026ct2\u003dus\u0026fd\u003dS\u0026url\u003dhttp://www.youredm.com/2015/07/19/paris-hilton-thinks-shes-proven-her-haters-wrong-in-recent-interview/\u0026cid\u003d52778904372071\u0026ei\u003dYQWvVbCBJPCY2gXJnqPYAQ\u0026usg\u003dAFQjCNFe7klvtDCPkOmE49vzKdLEVTWTKQ","language":"en","image":{"url":"http://www.youredm.com/wp-content/uploads/2013/11/paris-hilton-dj-lingerie-po.jpg?3f6961","tbUrl":"http://t0.gstatic.com/images?q\u003dtbn:ANd9GcTXULpHrDg4HdEg_SyY5XvvwnL9v0_RqsPcAaFo3h752eo2GQNQLF41IhA","originalContextUrl":"http://www.youredm.com/2015/07/19/paris-hilton-thinks-shes-proven-her-haters-wrong-in-recent-interview/","publisher":"Your EDM","tbWidth":80,"tbHeight":53},"relatedStories":[{"unescapedUrl":"http://www.dailymail.co.uk/tvshowbiz/article-3166903/Paris-Hilton-makes-entrance-skintight-feathered-gown-Ibiza-club-night.html","url":"http%3A%2F%2Fwww.dailymail.co.uk%2Ftvshowbiz%2Farticle-3166903%2FParis-Hilton-makes-entrance-skintight-feathered-gown-Ibiza-club-night.html","title":"\u003cb\u003eParis Hilton\u003c/b\u003e in skintight see-through feathered gown for Ibiza club night \u003cb\u003e...\u003c/b\u003e","titleNoFormatting":"Paris Hilton in skintight see-through feathered gown for Ibiza club night ...","location":"","publisher":"Daily Mail","publishedDate":"Sat, 18 Jul 2015 23:42:35 -0700","signedRedirectUrl":"http://news.google.com/news/url?sa\u003dT\u0026ct2\u003dus\u0026fd\u003dS\u0026url\u003dhttp://www.dailymail.co.uk/tvshowbiz/article-3166903/Paris-Hilton-makes-entrance-skintight-feathered-gown-Ibiza-club-night.html\u0026cid\u003d52778904372071\u0026ei\u003dYQWvVbCBJPCY2gXJnqPYAQ\u0026usg\u003dAFQjCNEItk5NezwgNjkjXZoI2MX5caES4A","language":"en"},{"unescapedUrl":"http://timesofindia.indiatimes.com/entertainment/english/hollywood/news/Paris-Hilton-wows-in-Ibiza/articleshow/48146046.cms","url":"http%3A%2F%2Ftimesofindia.indiatimes.com%2Fentertainment%2Fenglish%2Fhollywood%2Fnews%2FParis-Hilton-wows-in-Ibiza%2Farticleshow%2F48146046.cms","title":"\u003cb\u003eParis Hilton\u003c/b\u003e wows in Ibiza - The Times of India","titleNoFormatting":"Paris Hilton wows in Ibiza - The Times of India","location":"","publisher":"Times of India","publishedDate":"Mon, 20 Jul 2015 11:22:30 -0700","signedRedirectUrl":"http://news.google.com/news/url?sa\u003dT\u0026ct2\u003dus\u0026fd\u003dS\u0026url\u003dhttp://timesofindia.indiatimes.com/entertainment/english/hollywood/news/Paris-Hilton-wows-in-Ibiza/articleshow/48146046.cms\u0026cid\u003d52778904372071\u0026ei\u003dYQWvVbCBJPCY2gXJnqPYAQ\u0026usg\u003dAFQjCNEwhXa-5JiAoh0CgerF0D5a7b6H8Q","language":"en"},{"unescapedUrl":"http://thebanginbeats.com/2015/07/20/paris-hilton-believes-she-has-proven-her-haters-wrong/","url":"http%3A%2F%2Fthebanginbeats.com%2F2015%2F07%2F20%2Fparis-hilton-believes-she-has-proven-her-haters-wrong%2F","title":"\u003cb\u003eParis Hilton\u003c/b\u003e Believes She Has Proven Her Haters Wrong","titleNoFormatting":"Paris Hilton Believes She Has Proven Her Haters Wrong","location":"","publisher":"The Bangin Beats","publishedDate":"Mon, 20 Jul 2015 07:03:45 -0700","signedRedirectUrl":"http://news.google.com/news/url?sa\u003dT\u0026ct2\u003dus\u0026fd\u003dS\u0026url\u003dhttp://thebanginbeats.com/2015/07/20/paris-hilton-believes-she-has-proven-her-haters-wrong/\u0026cid\u003d52778904372071\u0026ei\u003dYQWvVbCBJPCY2gXJnqPYAQ\u0026usg\u003dAFQjCNGeA2p7zfO85SCgDSUXIjU-IMDtwQ","language":"en"}]}],"cursor":{"pages":[{"start":"0","label":1},{"start":"4","label":2},{"start":"8","label":3},{"start":"12","label":4},{"start":"16","label":5},{"start":"20","label":6},{"start":"24","label":7},{"start":"28","label":8}],"estimatedResultCount":"7432895","currentPageIndex":0,"moreResultsUrl":"http://news.google.com/nwshp?oe\u003dutf8\u0026ie\u003dutf8\u0026source\u003duds\u0026q\u003dParis+Hilton\u0026hl\u003den\u0026start\u003d0"}}, "responseDetails": null, "responseStatus": 200}
